doc/planning_scene_python/src/planning_scene_tutorial.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+import sys
+
+import moveit_commander
+import numpy as np
+import rospy
+from geometry_msgs.msg import PoseStamped
+import moveit.core
+from moveit.core import kinematic_constraints, collision_detection, robot_model
+from moveit.core.planning_scene import PlanningScene
+
+
+def main():
+    rospy.init_node("planning_scene_tutorial")
+    moveit_commander.roscpp_initialize(sys.argv)
+
+    # BEGIN_TUTORIAL
+    #
+    # Setup
+    # ^^^^^
+    #
+    # The :moveit_core_python:`core.planning_scene.PlanningScene` class can be easily setup and
+    # configured using a URDF and
+    # SRDF. This is, however, not the recommended way to instantiate a
+    # PlanningScene. At the time of writing there are not yet python bindings
+    # for the PlanningSceneMonitoir, which is is the recommended method to
+    # create and maintain the current planning scene
+    # using data from the robot's joints and the sensors on the robot. In
+    # this tutorial, we will instantiate a PlanningScene class directly,
+    # but this method of instantiation is only intended for illustration.
+
+    urdf_path = "/opt/ros/noetic/share/moveit_resources_panda_description/urdf/panda.urdf"
+    srdf_path = "/opt/ros/noetic/share/moveit_resources_panda_moveit_config/config/panda.srdf"
```

Comment:
This is a brittle way to load resoruces in ROS. Use the resource_load (I think that's the name) to resolve package names, without hard coding paths to someone's specific computer

Reply:
do you mean this? http://wiki.ros.org/resource_retriever
or just the rospkg python API? I think rospkg would work fine, assuming the person has the `moveit_resources_panda_*` repos available